### PR TITLE
fix leader-elect-resource-lock's description

### PR DESCRIPTION
**Pull request by malc0lm**
**What this PR does / why we need it**:
    The leader-elect-resource-lock description miss a space in pkg/client/leaderelectionconfig/config.go. It will effect other component's help document which use leader election.
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
    I think it's is needless to open a issue.
**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by malc0lm** [comment]
@caesarxuchao @deads2k Please review this PR, thanks.

File: pkg/client/leaderelectionconfig/config.go [hunk 1]
```diff
@@ -61,6 +61,6 @@ func BindFlags(l *componentconfig.LeaderElectionConfiguration, fs *pflag.FlagSet
 		"The duration the clients should wait between attempting acquisition and renewal "+
 		"of a leadership. This is only applicable if leader election is enabled.")
 	fs.StringVar(&l.ResourceLock, "leader-elect-resource-lock", l.ResourceLock, ""+
-		"The type of resource resource object that is used for locking during"+
+		"The type of resource resource object that is used for locking during "+
```
Comment (deads2k) [inline pkg/client/leaderelectionconfig/config.go]: fix "resource resource" while you're here?

Comment (malc0lm) [inline pkg/client/leaderelectionconfig/config.go]: No, I just added a space for 'during' and ''leader".  Or the help document will like this "The type of resource resource object that is used for locking duringleader election...."

Comment (deads2k) [inline pkg/client/leaderelectionconfig/config.go]: Can you fix the other typo on this line ("resource resource" ) in this commit.

**Comment by deads2k** [comment]
lgtm.  please squash for merge.

File: pkg/client/leaderelectionconfig/config.go (hunk 1 above)
Comment (malc0lm) [inline pkg/client/leaderelectionconfig/config.go]: Ok, I will squash my commits.

**Comment by malc0lm** [comment]
Thanks @deads2k , I have squashed my commits. Ready to be merged.

**Pull request merged**